ci-operator/step-registry/ibmcloud/deprovision/iamservice/minimal-permission/ibmcloud-deprovision-iamservice-minimal-permission-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#pre create the iam service-id and based it create the iam service api key for ibmcloud-ipi-minimal-permission test
+# based on doc https://docs.openshift.com/container-platform/4.14/installing/installing_ibm_cloud_public/installing-ibm-cloud-account.html#required-access-policies-ibm-cloud_installing-ibm-cloud-account
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    export region
+    echo "Try to login..."
+    "${IBMCLOUD_CLI}" config --check-version=false
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+#####################################
+##############Initialize#############
+#####################################
+ibmcloud_login
+
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+serviceName="${CLUSTER_NAME}-perm"
```

Comment:
We should save the service ID when it is being provisioned, then get it from there.